### PR TITLE
Ros2 add launch and remappings

### PR DESCRIPTION
**Pull request by delihus**
Added dynamically loaded components with their plugins in Gazebo Ignition.

### CODE REVIEW

File: launch/gz_components.launch.py [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+
+        if component["type"] == "LDR13":
+            actions.append(get_launch_description("ouster_os", package, namespace, component))
+
+        if component["type"] == "LDR20":
+            actions.append(get_launch_description("velodyne", package, namespace, component))
+
+        if component["type"] == "CAM01":
+            actions.append(get_launch_description("orbbec_astra", package, namespace, component))
+
```
Comment (rafal-gorecki) [inline launch/gz_components.launch.py:69]: Make it easier to maintain
```suggestion
    component_to_launch = {
        "LDR01": "slamtec_rplidar",
        "LDR06": "slamtec_rplidar",
        "LDR13": "ouster_os",
        "LDR20": "velodyne",
        "CAM01": "orbbec_astra"
    }

    for component in node["components"]:
        component_type = component["type"]
        if component_type in component_to_launch:
            launch_description = get_launch_description(component_to_launch[component_type], package, namespace, component)
            actions.append(launch_description)
```

File: launch/gz_orbbec_astra.launch.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    if device_namespace == "":
+        device_namespace_ext = ""
+
+    tf_prefix_ext = tf_prefix + "_"
+    if tf_prefix == "":
+        tf_prefix_ext = ""
+
+    parent_frame = tf_prefix_ext + camera_name + "_depth_optical_frame"
+    child_frame = (
+        "panther/base_link//"
```
Comment (rafal-gorecki) [inline launch/gz_orbbec_astra.launch.py:42]: Should be `//`?

Comment (delihus) [reply launch/gz_orbbec_astra.launch.py:42]: Yup. This fixes the bug.

File: launch/gz_components.launch.py (hunk 1 above)
Comment (delihus) [reply launch/gz_components.launch.py]: Nah, I prefer `get_launch_description` to stay.

File: test/component.urdf.xacro [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  <xacro:arg name="components_config_path" default="None" />
+  <xacro:property name="components_config_path_property" value="$(arg components_config_path)" />
+  <xacro:unless value="${components_config_path_property == 'None'}">
+    <xacro:include filename="$(find ros_components_description)/urdf/components.urdf.xacro"
+      ns="husarion_components" />
+
+    <xacro:husarion_components.create_components
+      components_config_path="${components_config_path_property}"
+      use_gpu="True"
+      simulation_engine="ignition-gazebo"
```
Comment (rafal-gorecki) [inline test/component.urdf.xacro:14]: Probably unncecesary
```suggestion
      simulation_engine="ignition-gazebo"
```

File: launch/gz_components.launch.py
[the remark this answers is not in the document]
Comment (delihus) [reply launch/gz_components.launch.py:1]: So. We have changed the idea of our urdf. Changes applied in this pr.

**Pull request merged**